kodilla-testing/src/main/java/com/kodilla/testing/forum/ForumComment.java

```diff
@@ -0,0 +1,44 @@
+package com.kodilla.testing.forum;
+
+import java.util.Objects;
+
+public class ForumComment {
+    ForumPost forumPost;
+    String commentBody;
+    String author;
+
+    public ForumComment(ForumPost forumPost, String commentBody, String author) {
```

Comment:
a co to za klasa?

Reply:
Klasy ForumUser, ForumPost, ForumComment i ForumTestSuite były tworzone jako przykłady w trakcie kursy